orttraining/orttraining/test/python/orttraining_test_ortmodule_mnist_deepspeed.py

```diff
@@ -0,0 +1,228 @@
+"""Test for a simple ORTModule using the high-level DeepSpeed API.
+
+To run on the local GPU(s):
+
+```
+$ pip install deepspeed
+$ deepspeed orttraining_test_ortmodule_mnist_deepspeed.py \
```

Comment:
nit: `python orttraining_test_ortmodule....'?

Reply:
No, this is using the `deepspeed` launcher binary that is added to your `$PATH` when you install DeepSpeed. It sets various environment variables and flags used in the test.

Reply:
OK.